packages/generator/README.md

```diff
@@ -45,8 +46,9 @@ const { ClientConfig, helpers, Search } = CommerceSdk;
 // const { ClientConfig, helpers, Search } = require("commerce-sdk");
 
 // Create a configuration to use when creating API clients
-// In TypeScript, let config = new ClientConfig();
-const config = {
+let config = new ClientConfig();
```

Comment:
Why is the `ClientConfig` instance immediately discarded?

Reply:
Fixed